kolibri/plugins/pwa/templates/sw.js

```diff
@@ -0,0 +1,8 @@
+{% load i18n core_tags webpack_tags content_tags cache %}
+
+const VERSION = '{{ version }}';
+
+/* Stub ServiceWorker to keep Chrome happy. */
+self.addEventListener('install', (event) => {
+    console.log('[SW] Installing ServiceWorker version:', VERSION);
```

Comment:
Not important, but I think in most developer consoles, console.logs from the service worker will be prefixed anyway.